WebRoot/css/zm.css

```diff
@@ -7279,4 +7269,78 @@ input[type="radio"] + label {
 
 .ZmWhiteBlackViewNumberUsed {
 	@WhiteBlackListViewItemsNumberUsed@
-}
\ No newline at end of file
+}
+
+/* Preference signature interface panel */
+.ZmPrefSignaturePage {
+	@PrefSignaturePage@
+}
+
+.ZmPrefSignaturePage hr {
+	@PrefSignaturePageSeparator@
+}
+
+.ZmSignatureNameWrapper {
+	@PrefSignatureNameWrapper@
+}
+
+.ZmSignatureNameWrapper > label {
+	@PrefSignatureNameLabel@
+}
+
+.ZmSignatureNameWrapper .DwtInputField input[type='text'] {
+	@PrefSignatureNameInput@
+}
+
+.ZmSignatureFormatWrapper {
+	@PrefSignatureFormatWrapper@
+}
+
+.ZmSignatureListWrapper {
+	@PrefSignatureListWrapper@
+}
+
+.ZmSignatureListView {
+	@PrefSignatureListView@
+}
+
+.ZmSignatureListView [class*='Row'] TD {
```

Comment:
There are many classes not just Row, like Row-focussed, Row-disabled etc. For all such cases we need same styling. So instead of targeting each one separately, this works out well.